modules/gapi/samples/pipeline_modeling_tool/pipeline_builder.hpp

```diff
@@ -439,8 +533,36 @@ Pipeline::Ptr PipelineBuilder::construct() {
             // (3). Extract proto input from every input node.
             inputs.push_back(in_data.arg.value());
         }
+        // NB: If node shouldn't be called on each iterations,
+        // it should be wrapped into subgraph which is able to skip calling.
+        if (call.call_every_nth != 1u) {
+            // FIXME: Limitation of the subgraph operation (<GMat(GMat)>).
+            // G-API doesn't support dynamic number of inputs/outputs.
+            if (inputs.size() > 1u) {
+                throw std::logic_error(
+                        "skip_frame_nth is supported only for single input subgraphs");
```

Comment:
printing of `inputs.size()` will help in troubleshooting

Reply:
Good point.

Reply:
Done